WcaOnRails/config/locales/fr.yml

```
@@ -1350,6 +1350,7 @@ fr:
       manage_app: Gérer vos applications
       #original_hash: 0e8b212
       manage_auth_app: Gérer vos applications authentifiées
+      media_submission: Soumission aux médias
```

lol :)
I think this would fit better in the context:
```suggestion
      media_submission: Partager des photos et vidéos
```